docs/Wallets/list.md

```diff
@@ -11,20 +11,21 @@ Note that in case of issues, usually only queries relating to official wallets c
+|[Flint]      |Light    |9          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:                    |( :x:,:ballot_box_with_check:,:ballot_box_with_check: ) |[Partial](https://github.com/dcSpark)                    |
+|[Typhon]     |Light    |9          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |( :x:,:ballot_box_with_check:,:ballot_box_with_check: ) |[Partial](https://github.com/StricaHQ)|
+|[ADAlite]    |Light    |9          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |( :x:,:x:,:ballot_box_with_check: )                     |[:ballot_box_with_check:](https://github.com/vacuumlabs/adalite)|
+|[NuFi]       |Light    |8          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |( :x:,:ballot_box_with_check:,:ballot_box_with_check: ) |:x:                    |
+|[Daedalus]   |Full     |8          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |( :ballot_box_with_check:,:x:,:x: )                     |[:ballot_box_with_check:](https://github.com/input-output-hk/daedalus)|
+|[Nami]       |Light    |7          |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |( :x:,:ballot_box_with_check:,:x: )                     |[:ballot_box_with_check:](https://github.com/Berry-Pool/nami-wallet)|
+|[Gero]       |Light    |7          |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |( :x:,:ballot_box_with_check:,:x: )                     |:x:                    |
+|[LodeWallet] |Light    |6          |:ballot_box_with_check:|:ballot_box_with_check:|:x:                    |:x:                    |:x:                    |:x:                    |:ballot_box_with_check:|:ballot_box_with_check:|:ballot_box_with_check:|( :x:,:ballot_box_with_check:,:x: )                     |:x:                    |
```

Comment:
looks like there was accidentally a carriage return instead of being on single line

Reply:
What do you mean?  All the lodewallet info seems to be on line 25.  And grepping for `\r`brings up nothing, so this file shouldn't have any carriage returns.  I must be misunderstanding.

Reply:
You're right, my bad - copying the line messed up in mobile, as it was showing up wrapped text due to next fields...false alarm